src/js/bootstrap-slider.js

```diff
@@ -1261,6 +1269,11 @@
 
 				return true;
 			},
+			_touchstart: function(ev) {
+				var touch = ev.changedTouches[0];
```

Comment:
same feedback here as below (Could you add a check in your code for when `ev.changedTouches[0]` is `undefined`?)